client/src/components/aggregations/ReportsMapWidget.tsx

```diff
@@ -35,8 +37,9 @@ const ReportsMapWidget = ({
     const markerArray = []
     values.forEach(report => {
       if (Location.hasCoordinates(report.location)) {
-        let label = `<a href="${Report.pathFor(report)}" target="_blank">${_escape(report.intent || "<undefined>")}</a>` // escape HTML in intent!
-        label += `<br/>@ <b>${_escape(report.location.name)}</b>` // escape HTML in locationName!
+        let label = `<b>Report:</b> <a href="${Report.pathFor(report)}" target="_blank">${_escape(report.intent || "<undefined>")}</a>` // escape HTML in intent!
+        label += `<br/><b>Location:</b> ${_escape(report.location.name)}` // escape HTML in locationName!
+        label += `<br/><b>Date:</b> ${_escape(moment(report.engagementDate).format(Settings.dateFormats.forms.displayShort.withTime))}`
```

Comment:
You should check whether engagements actually _have_ a time; so for consistency with other places, you should call `Report.getEngagementDateFormat()`:
```javascript
        label += `<br/><b>Date:</b> ${moment(report.engagementDate).format(Report.getEngagementDateFormat())}`
```
(note there's in principle also no need for _escape)